phys2bids/tests/test_integration.py

```diff
@@ -0,0 +1,466 @@
+import os
+import wget
+import glob
+import json
+import subprocess
+
+from pkg_resources import resource_filename
+
+from phys2bids.phys2bids import phys2bids
+from phys2bids._version import get_versions
+
+
+def test_logger():
+    """
+    Tests the logger
+    """
+
+    # Move into folder
+    test_path = resource_filename('phys2bids', 'tests/data')
+    subprocess.run(f'cd {test_path}', shell=True, check=True)
+
+    # Input arguments
+    test_filename = 'tutorial_file.txt'
+    test_chtrig = 1
+    test_outdir = test_path
+
+    # Phys2bids call through terminal
+    subprocess.run(f'phys2bids -in {test_filename} -indir {test_path} '
+                   f'-chtrig {test_chtrig} -outdir {test_outdir}', shell=True, check=True)
+
+    # Read logger file
+    files = os.listdir(test_path)
+    logger_idx = [log_idx for log_idx, log_str in enumerate(
+                      files) if 'phys2bids' in log_str]
+    logger_file = files[logger_idx[0]]
+
+    with open(os.path.join(test_path, logger_file)) as logger_info:
+        logger_info = logger_info.readlines()
+
+    # Get version info
+    version_idx = [log_idx for log_idx, log_str in enumerate(
+                      logger_info) if 'phys2bids version' in log_str]
+    version_found = logger_info[version_idx[0]]
+    current_version = get_versions()
+
+    # Checks logger version is the current version
+    assert current_version['version'] in version_found
+
+    # Removes generated files
+    os.remove(os.path.join(test_path, logger_file))
+
+
+def test_integration_tutorial():
+    """
+    Does an integration test with the tutorial file
+    """
+    test_path = resource_filename('phys2bids', 'tests/data')
+    test_filename = 'tutorial_file.txt'
+    test_full_path = os.path.join(test_path, test_filename)
+    test_chtrig = 1
+    test_outdir = test_path
+    test_ntp = 158
+    test_tr = 1.2
+    test_thr = 0.735
+    phys2bids(filename=test_full_path, chtrig=test_chtrig, outdir=test_outdir,
+              num_timepoints_expected=test_ntp, tr=test_tr, thr=test_thr)
+
+    # Check that files are generated
+    assert os.path.isfile(os.path.join(test_path, 'tutorial_file.log'))
+    assert os.path.isfile(os.path.join(test_path, 'tutorial_file.json'))
+    assert os.path.isfile(os.path.join(test_path, 'tutorial_file.tsv.gz'))
+    assert os.path.isfile(os.path.join(test_path, 'tutorial_file_trigger_time.png'))
```

Comment:
What about a for-loop here? E.g.,

```suggestion
    # Check that files are generated
    for suffix in ['.log', '.json', '.tsv.gz', '_trigger_time.png']:
        assert os.path.isfile(os.path.join(test_path, 'tutorial_file' + suffix))
```